### ISSUE TITLE
Not getting accuracy on Custom Dataset of DRRG Model

### ISSUE THREAD

**Issue by payal211** [open-mmlab/mmocr#935]
HI,

Earlier I had train Text Detection Model on Custom Dataset using DRRG model on single GPU, and I able to achieve good result only at 100th epoch.
Here I am attaching log of that training
[20211215_035044.log](https://github.com/open-mmlab/mmocr/files/8487333/20211215_035044.log)

Now to speed up the training, I started distributed training with 4 GPUs, and I completed 1200 epochs But not able to achieve result.
here is the attached log file:
[20220408_122038.log](https://github.com/open-mmlab/mmocr/files/8487355/20220408_122038.log)

After getting not accuracy I continuing the training with same data from 1200th epoch to another 1200 epoch, the loss is continuously decreasing but the issue remains same, not able to detect Text
here is the attached log file:
[20211215_035044.log](https://github.com/open-mmlab/mmocr/files/8487388/20211215_035044.log)

**Comment by gaotongxiao** [comment]
As you change the number of GPUs to use, the suggested learning rate is usally scaled up/down accordingly as the actual batch size has also been changed. It seems you haven't done this even though you were getting 4x larger batch size. I'd suggest you restart the training from scratch with a new learning rate. And you can read [it](https://www.baeldung.com/cs/learning-rate-batch-size) for more information.
